axelrod/strategies/probabilistic_hill_climb.py

```diff
@@ -0,0 +1,55 @@
+from axelrod.action import Action
+from axelrod.player import Player
+
+C, D = Action.C, Action.D
+
+class ProbabilisticHillClimb(Player):
+	"""
+	Defects with probability of 50%.
```

Comment:
This strategy defects with a given probability that changes over time. It's initial probability of defection is 0.5.